consensus/src/persistent_liveness_storage.rs

```diff
@@ -127,18 +139,27 @@ impl LedgerRecoveryData {
             .find(|qc| qc.certified_block().id() == root_block.id())
             .ok_or_else(|| format_err!("No QC found for root: {}", root_id))?
             .clone();
-        let root_ordered_cert = quorum_certs
-            .iter()
-            .find(|qc| qc.commit_info().id() == root_block.id())
-            .ok_or_else(|| format_err!("No LI found for root: {}", root_id))?
-            .clone();
 
+        let (root_ordered_cert, root_commit_cert) = if order_vote_enabled {
```

Comment:
Do we have crash recovery test to cover this change?

Reply:
We don't have seem to have any crash recovery test in unit tests. IIUC testing crash recovery should be part of end to end testing. Forge realistic_env tests add some chaos. cc @zekun000 Is there a better way to test this?